nuitka/gui/TreeDisplay.py

```diff
@@ -197,9 +198,9 @@ def check(item):
         return check(self.root_item)
 
 
-class InspectNodeTreeDialog(QtGui.QDialog):
+class InspectNodeTreeDialog(QtWidgets.QDialog):
```

Comment:
Does the GUI work with that now or why did you change this?

Reply:
No, the gui still doesn't work yet, I need to fix it but i wanted the pr to get merged first so i'd fix the gui on another smaller PR. It wasn't working this change btw and that QtGui isn't importing QDialog on pyqt5 anyway.